src/components/Challenge/index.js

```diff
@@ -0,0 +1,35 @@
+import {useNavigate} from 'react-router-dom';
+
+import {useStore} from '../../hooks/useStore';
+import Button from '../Button/index';
+import {ChallengeContainer} from '../Challenge/styled';
+
+export default function Challenge({
+	challengeId,
+	image,
+	title,
+	startingPoint,
+	endingPoint,
+	distance,
+}) {
+	const navigate = useNavigate();
+	const setActivChallengeId = useStore(state => state.setActivChallengeId);
+	return (
+		<ChallengeContainer>
+			<img src={image} aria-label={title} alt={title} />
```

Comment:
If you have an `alt`-attribute I guess you do not need the `aria-label` here - as they are doing basically the same :)